Processor.php

```diff
@@ -60,7 +74,26 @@ public function processFile(array $config)
             mkdir($dir, 0755, true);
         }
 
-        file_put_contents($realFile, "# This file is auto-generated during the composer install\n" . Yaml::dump($actualValues, 99));
+        if ('yml' == $fileExtension) {
+            $output = <<<EOF
+# This file is auto-generated during the composer install
+%actual.values%
+EOF
+            ;
+            $output = str_replace('%actual.values%', Yaml::dump($actualValues, 99), $output);
```

Comment:
If the `$fileExtension` is `yml` then the string concatenation will be better, but for `php` it's less code and looks more readable this approach, I've just keep the same pattern on both, should I change and PR again?